### PR TITLE
Resubmit GA

### PR DESCRIPTION
**Pull request by askpatrickw**
Resubmit of GA elections.toml

I could not find some information on GA site, I think some is behind a login for registered voters. Got some info from Ballotpedia.

Purposefully skipped the Run-Off Absentee info

### CODE REVIEW

**Comment by tannewt** [comment]
@theacodes Do you know the deadlines for applying for absentee?

File: states/georgia/elections.toml [hunk 1]
```diff
@@ -3,17 +3,78 @@ date = 2020-06-09
...
-name = "General"
+name = "General Election and Special Election"
 original_date = 2020-11-03
 
+[20201103.registration]
+online_by = 2020-10-05
+received_by = 2020-10-05
+sources = ["https://sos.ga.gov/admin/files/2020%20Revised%20Short%20Calendar.pdf"]
+
+[20201103.absentee.application]
```
Comment (askpatrickw) [inline states/georgia/elections.toml:52]: This needs a re-review... might be 11/2 since you can do it in-person.

**Comment by theacodes** [comment]
I unfortunately have no idea what the deadline is. :(

**Comment by askpatrickw** [comment]
I'll ask...

```
To: elections.voterregistration@fultoncountyga.gov

Hey,

When is deadline to signup for an absentee ballot?
And is that deadline the same for all counties?

I looked on the the web site and couldn't find this answer:
https://sos.ga.gov/index.php/Elections/absentee_voting_in_georgia

Thanks so much,
Patrick
```

I even threw in a "hey".. maybe that's only a NC thing though. My parent's are from there.. ;-)

**Comment by tannewt** [comment]
@askpatrickw want to merge this in and follow up with absentee dates later? Maybe open an issue?

**Comment by askpatrickw** [comment]
I was hoping they'd answer quickly... I'll review, comment out anything which can't be confirmed, submit for review and open an issue for anything which needs follow-up.

Good call!

**Comment by askpatrickw** [comment]
On re-read I think what they are saying is you want to vote absentee in person? We have early voting, go do that.
I removed the in_person from absentee.application and padded the other dates.

File: states/georgia/elections.toml [hunk 2]
```diff
@@ -20,9 +20,8 @@ received_by = 2020-10-05
 sources = ["https://sos.ga.gov/admin/files/2020%20Revised%20Short%20Calendar.pdf"]
 
 [20201103.absentee.application]
-email_by = 2020-11-03
-in_person_by = 2020-11-03
-received_by = 2020-11-03
+email_by = 2020-10-20 # Padded 14 Days
+received_by = 2020-10-20 # Padded 14 Days
```
Comment (tannewt) [inline states/georgia/elections.toml:54]: I think you want fax and in_person still. "Absentee ballot applications can be returned by mail, fax, email (as an attachment), or in-person to the local County Board of Registrar’s Office."

**Review by tannewt — CHANGES_REQUESTED** [review]

File: states/georgia/elections.toml (hunk 2 above)
Comment (askpatrickw) [reply states/georgia/elections.toml:54]: 👍

File: states/georgia/elections.toml [hunk 3]
```diff
@@ -3,17 +3,79 @@ date = 2020-06-09
...
 
-[20200811]
-date = 2020-08-11
-name = "Congressional Primary Run-off (if needed)"
-original_date = 2020-08-11
+# [20200811]
+# date = 2020-08-11
+# name = "Congressional Primary Run-off (if needed)"
+# original_date = 2020-08-11
+# There is no run-off this election cycle
```
Comment (tannewt) [inline states/georgia/elections.toml]: It looks like it is happening: https://sos.ga.gov/index.php/elections/last_day_to_register_for_federal_and_state_runoff_elections Do you want to add it?

Comment (askpatrickw) [reply states/georgia/elections.toml]: added the runoff

**Review by tannewt — APPROVED** [review]
Thank you!

**Pull request merged**